config/default.schema.json

```diff
@@ -207,6 +207,11 @@
 			"type": "string",
 			"description": "Synchronous encryption key used to secure storage provider secrets. Only used with FEATURE_MULTIPLE_S3_PROVIDERS_ENABLED=true",
 			"pattern": ".{16}.*"
+		},
+		"SENTRY_SAMPLE_RATE": {
+			"type": "number",
+			"default": 0.10,
```

Comment:
i would suggest to keep 1.0 as default due normally all requests should be catched if nothing different has been defined. please move SENTRY_DSN from globals to config.